scripts/exglobal_prep_ocean_obs.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
+# exglobal_prep_ocean_obs.py
+# Pepares observations for coupled ocean DA
```

Comment:
for marine DA instead of coupled ocean DA

Reply:
changed